bmq-sdk/src/test/java/com/bloomberg/bmq/impl/infr/util/expressionvalidator/TestExpressionValidator.java

```diff
@@ -0,0 +1,360 @@
+                    {"aaa_111BBBaaa222__BBBaaa3_3_3BBB > 0", true, null},
+                    {"B_a_1_B_a_2_B_a_3_B_a_4_B_a_5_B_ > 0", true, null},
+
+                    // readable examples
+                    {"camelCase > 0", true, null},
+                    {"snake_case > 0", true, null},
+                    {"PascalCase > 0", true, null},
+                    {"price > 0", true, null},
+                    {"BetterLateThanNever > 0", true, null},
+                    {"firmId > 0", true, null},
+                    {"TheStandardAndPoor500 > 0", true, null},
+                    {"SPX_IND > 0", true, null},
+
+                    // all available characters
+                    {"abcdefghijklmnopqrstuvwxyz_0123456789 > 0", true, null},
+                    {"ABCDEFGHIJKLMNOPQRSTUVWXYZ_0123456789 > 0", true, null},
+
+                    // negative examples
+                    {"0name > 0", false, "syntax error, missed operation at offset 1"},
+                    {"1NAME > 0", false, "syntax error, missed operation at offset 1"},
+                    {"23456789Name > 0", false, "syntax error, missed operation at offset 8"},
+                    {
+                        "_nameName > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 0"
+                    },
+                    {
+                        "0_NameName > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 1"
+                    },
+                    {
+                        "_1n > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 0"
+                    },
+                    {
+                        "1_N > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 1"
+                    },
+                    {
+                        "_ > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 0"
+                    },
+                    {
+                        "_11111111111111111111111111111 > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 0"
+                    },
+                    {"22222222222222222222222222222_ > 0", false, "integer overflow at offset 0"},
+                });
+    }
+
+    @Test
+    public void expressionValidatorTest() {
+        java.io.Reader reader = new java.io.StringReader(inputExpression);
+        ExpressionValidator validator = new ExpressionValidator();
+        try {
+            boolean result = validator.validate(reader);
+            assertEquals(result, expectedResult);
+            assertEquals(validator.getErrorMessage(), expectedErrorMessage);
+        } catch (java.io.IOException e) {
```

Comment:
No need to catch exception unless want to handle, log etc.
`IOException` may be declared as a checked exception for the test method

Reply:
Got rid of `catch`, declared `IOException` as a checked exception